apio-consumer/src/main/kotlin/com/liferay/apio/consumer/delegates/ObserveDelegate.kt

```diff
@@ -24,27 +24,27 @@ fun <T> observe(onChange: (T?) -> Unit): ReadWriteProperty<Any, T?> = ObserveDel
 @PublishedApi
 internal class ObserveNonNullDelegate<T>(val onChange: (T) -> Unit) : ReadWriteProperty<Any, T?> {
 
-	private var t: T? = null
+    private var t: T? = null
 
-	override fun getValue(thisRef: Any, property: KProperty<*>): T? = t
+    override fun getValue(thisRef: Any, property: KProperty<*>): T? = t
 
-	override fun setValue(thisRef: Any, property: KProperty<*>, value: T?) {
-		t = value
+    override fun setValue(thisRef: Any, property: KProperty<*>, value: T?) {
+        t = value
```

Comment:
this code looks pretty similar, right?

Reply:
oops, just a bad SF, the @dgarciasarai commits solves that.